apps/core/factories.py

```diff
@@ -1,10 +1,18 @@
 from factory import Faker, Iterator, SubFactory
 from factory.django import DjangoModelFactory
+from django.utils import timezone
 
 from apps.data.factories import EntryFactory, RepositoryFactory
+from faker import Faker as faker_Faker
```

Comment:
You don't need to re-import Faker, it's already imported via `factory` (see line 1).